js/interface.js

```diff
@@ -804,6 +804,12 @@
   attachObservers();
   // Load menu widgets on startup
   loadCustomMenuWidgets();
+
+  Fliplet.Studio.onMessage((eventData) => {
+    if (eventData && eventData.data && eventData.data.type === 'menu-tab-changed') {
+      $('.nav.nav-tabs li a[aria-controls="menu-manager"]').trigger('click');
```

Comment:
See https://getbootstrap.com/docs/3.3/javascript/#tabs

![image](https://user-images.githubusercontent.com/290733/168743846-7cf676a3-c691-4192-b468-a308efebcc6a.png)

This is how you use Bootstrap's API to select a tab by name. Please use the `href` target, not the ARIA accessibility attribute, and use the Bootstrap API.